### ISSUE TITLE
An error occurred starting screensaver. Action 'activate' failed. Ensure you have xscreensaver installed and running.

### ISSUE THREAD

**Issue by Raytle** [lxqt/lxqt#1994]
This is similar to issue #985.

I'm running Lubuntu 20.04 LTS

1. I navigated to Menu » Preferences » LXQt Settings » Session Settings » Autostart
2. I unchecked the check box next to Xscreensaver
3. I also ran `sudo apt-get remove xscreensaver`

Each time I run `systemctl suspend` the following popup message appears on my screen...

> An error occurred starting screensaver. Action 'activate' failed. Ensure you have xscreensaver installed and running.

I don't want that popup message to appear on my screen.

I realize Arch Linux is different than Ubuntu, however, please see the following issue for reference...

[Harmless popup, I would like to get rid of it](https://bbs.archlinux.org/viewtopic.php?id=231076)

**Comment by tsujan** [comment]
It's simple: somewhere, you've told the system to lock the screen before suspending, while there's no locker.

Since you have Lubuntu 20.04, your LXQt should be outdated (unless you've compiled it yourself) but see if `LXQt Session Settings → Lock screen before suspending/hibernating` exits;  then just uncheck it, log out and log in.

**Comment by tsujan** [comment]
The issue template is there for you *to read it*. It says, **"Dear users of stable and LTS (long term service) distributions: Please do NOT file bugs against old (dead) versions but use your distribution bugtracker instead."**

**Comment by Raytle** [comment]
> It's simple: somewhere, you've told the system to lock the screen before suspending, while there's no locker.

Here's the script I run..

```
i3lock -u -i '/home/y/Pictures/Wallpaper.png' &
sleep 4 &&
xtrlock &
sleep 2 &&
systemctl suspend
```

> Since you have Lubuntu 20.04, your LXQt should be outdated (unless you've compiled it yourself) but see if `LXQt Session Settings → Lock screen before suspending/hibernating` exits; then just uncheck it, log out and log in.

Thanks. I found that...

https://i.imgur.com/urBsShd.png

**Comment by Raytle** [comment]
> The issue template is there for you _to read it_. It says, **"Dear users of stable and LTS (long term service) distributions: Please do NOT file bugs against old (dead) versions but use your distribution bugtracker instead."**

I'm sorry. I actually did read it. Perhaps I was wrong but it seemed to me that my issue might not have been addressed by the fix in issue #985.

I appreciate your help and your time.
